opentracing-spring-web-autoconfigure/src/main/java/io/opentracing/contrib/spring/web/autoconfig/ServerTracingAutoConfiguration.java

```diff
@@ -25,11 +29,26 @@
 public class ServerTracingAutoConfiguration {
     private static final Logger log = Logger.getLogger(ServerTracingAutoConfiguration.class.getName());
 
+    @Autowired(required=false)
+    @Qualifier("opentracing.http.skipPattern")
+    private Set<String> skipPatterns;
```

Comment:
Problem is that autowiring can't happen in `WebTracingConfiguration` - that seems to be the role that the `ServiceTracingAutoConfiguration` is assuming - so it currently seems this class is the best place to autowire the `skip patterns` and then supply them via the builder to `WebTracingConfiguration` (as is currently done with the default static pattern).

However could change the skip pattern to be a type, rather than using a qualifier?